### DISCUSSION TITLE
Supported Output Formats of .xls

### DISCUSSION THREAD

**Discussion by parkAlly** [exceljs/exceljs#2530] [Ideas & Proposals]
how to get output file ext '.xls' ?

**Comment by Rycochet** [comment]
You can't - they're totally different things, and only the xml based .xlsx is supported.

**Comment by parkAlly** [comment]
@Rycochet  okey , thanks

**Comment by Divyeshkhatri** [comment]
If I want to add this feature in this library, what should I need?

Thanks.

**Comment by Rycochet** [comment]
@Divyeshkhatri A complete rewrite...

They're completely incompatible and unrelated formats - so to start with you'd need to research the binary file format used, then start looking into the internal data layout used, then write a translation layer between the xlsx (zipped xml) based storage used here and the format in xls - then finally write a binary translation output that can be used to create a file either in the browser or via NodeJS...

In other words... A complete rewrite.

**Comment by Divyeshkhatri** [comment]
@Rycochet awsome

**Comment by Ryan2128** [comment]
Is there any progress in this work？
@Siemienik

**Comment by Siemienik** [comment]
@SnowyEvening nope :(

**Comment by Ryan2128** [comment]
:(   An unfortunate news.
